test/framework/cloudstack.go

```diff
@@ -302,3 +302,14 @@ func (c *CloudStack) WithRedhat123() api.ClusterConfigFiller {
 		),
 	)
 }
+
```

Comment:
why is this needed?

Reply:
Ah, are you referring to the comment? My bad, i forgot to remove it